app/models/task.rb

```diff
@@ -17,7 +17,7 @@ class Task < ApplicationRecord
   ].freeze
 
   belongs_to :claim
-  belongs_to :created_by, class_name: "DfeSignIn::User"
+  belongs_to :created_by, class_name: "DfeSignIn::User", optional: true
```

Comment:
I thought we had changed this to no longer be optional. How do we track who creates tasks?
In an ideal world there would be a `system` type user

Reply:
Nope, that's not what we discussed (see previous).